live-examples/css-examples/logical-properties/border-inline-end-color.html

```diff
@@ -0,0 +1,42 @@
+<section id="example-choice-list" class="example-choice-list large" data-property="border-inline-end-color">
+    <div class="example-choice" initial-choice="true">
+        <pre><code class="language-css">border-inline-end-color: red; 
+writing-mode: horizontal-tb;</code></pre>
+        <button type="button" class="copy hidden" aria-hidden="true">
+            <span class="visually-hidden">Copy to Clipboard</span>
+         </button>
+    </div>
+    
+    <div class="example-choice">
+        <pre><code class="language-css">border-inline-end-color: #32a1ce; 
+writing-mode: vertical-rl;
+        </code></pre>
+        <button type="button" class="copy hidden" aria-hidden="true">
+            <span class="visually-hidden">Copy to Clipboard</span>
+        </button>
+    </div>
+    
+    <div class="example-choice">
+        <pre><code class="language-css">border-inline-end-color: rgb(170, 50, 220, .6);
+writing-mode: horizontal-tb;
+direction: rtl;</code></pre>
```

Comment:
Because this choice is 3 lines, this example (and all the `inline` ones I think) is going to overflow, which I would like to avoid.

I think I'd suggest only having 3 examples, and dropping the 4th one:

```
border-inline-end-color: hsl(60, 90%, 50%, .8);
writing-mode: vertical-lr;
```

...which is close enough to the second:

```
border-inline-end-color: #32a1ce; 
writing-mode: vertical-rl;
```

Does that sound OK to you?

Reply:
yup, I've updated the inline ones accordingly.